scrapy/utils/conf.py

```diff
@@ -108,3 +109,31 @@ def get_sources(use_closest=True):
     if use_closest:
         sources.append(closest_scrapy_cfg())
     return sources
+
+
+def process_uri_list(uri_list, settings):
+    """
+    Receives a list of URIs (most likely from the 'crawl' or 'runspider'
+    commands) and returns a dictionary of {URI: file format}
+    """
+    valid_output_formats = without_none_values(
+        settings.getwithbase('FEED_EXPORTERS')
+    ).keys()
+
+    out = {}
+    for element in uri_list:
+        try:
+            feed_uri, feed_format = element.rsplit(':', 1)
+        except ValueError:
+            feed_uri = element
+            feed_format = os.path.splitext(element)[1].replace(".", "")
+        else:
+            if feed_uri == '-':
+                feed_uri = 'stdout:'
+        if feed_format not in valid_output_formats:
+            raise UsageError("Unrecognized output format '%s', set one after a"
+                             " semicolon (i.e. -o <URI>:<FORMAT>) or as a file"
```

Comment:
```suggestion
                             " colon (i.e. -o <URI>:<FORMAT>) or as a file"
```

Reply:
True, I alwasys get colons and semicolons confused by the name.